dali/ft/filecopy.cpp

```diff
@@ -3000,26 +3014,22 @@ void FileSprayer::spray()
 bool FileSprayer::isSameSizeHeaderFooter()
```

Comment:
what is the role of isSameSizeHeaderFooter() ?
I see some code that sets FPheaderLength and FPfooterLength as meta properties of each part:
```
                if (!sameSizeHeaderFooter)
                {
                    FilePartInfo & curHeaderSource = sources.item(whichHeaderInput);
                    curProps.setPropInt64(FPheaderLength, curHeaderSource.xmlHeaderLength);

                    FilePartInfo & curFooterSource = sources.item(cur.whichInput);
                    curProps.setPropInt64(FPfooterLength, curFooterSource.xmlFooterLength);

                    if ( idx+1 != partition.ordinality() )
                        whichHeaderInput = partition.item(idx+1).whichInput;
                }
```

so if not same as 1st part, then sets into part,
but I don't think I see where these per part properties are used?

Reply:
@AttilaVamos - was there an answer to this question?

Reply:
It seems my previous answer went missing. 
This implementation was part of the https://track.hpccsystems.com/browse/HPCC-13331 more than 4 years ago. I think only the despray uses this information.

Reply:
Ok thanks.